src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { Card, Avatar, Typography, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { greyScale } from '../../style-library/types/theme.helpers';
+import { SIZE_CONFIGS, VARIANT_CONFIGS } from './ProfileCard.constants';
+
+import type {
+  StyledProfileCardProps,
+  ProfileCardSize,
+  ProfileCardVariant,
+} from './ProfileCard.types';
+
+export const StyledCard = styled(Card, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size }) => {
+  const sizeConfig = SIZE_CONFIGS[$size];
+
+  return {
+    padding: theme.spacing(sizeConfig.padding / 8),
+    borderRadius: theme.spacing(2),
+    boxShadow: theme.shadows[2],
+    transition: theme.transitions.create(['box-shadow', 'transform'], {
+      duration: theme.transitions.duration.short,
+    }),
+    '&:hover': {
+      boxShadow: theme.shadows[4],
+      transform: 'translateY(-2px)',
+    },
+    [theme.breakpoints.down('sm')]: {
+      padding: theme.spacing(2),
+    },
+    [theme.breakpoints.up('md')]: {
+      padding: theme.spacing(sizeConfig.padding / 8),
```

Comment:
Same pattern here: padding on md uses `theme.spacing(sizeConfig.padding / 8)`. Use theme units stored in constants directly instead of dividing by 8.

**Sugerencia:**
```typescript
As above: store paddingUnits in constants and use `theme.spacing(sizeConfig.paddingUnits)`.
```